### PR TITLE
fix bug when comparing nonexistent columns

### PR DESCRIPTION
**Pull request by fschloesser**
We talked about this. It seems pandas can cope better with nonexisting data.

### CODE REVIEW

**Comment by GregorCH** [comment]
Hm, I am not convinced. These concat-operations look unnecessarily expensive (they create a new data frame object that gets directly reduced afterwards) to me. Usually, I would say, what does it matter,  but currently, we try to make it faster.

The fastest thing to do would be to say: "I do not know what this minimum/maximum value is supposed to be, because it is not a column, and not an integer or float, either." 

I would vote for printing a warning in this case and continue, because errors because some data is missing are hard to fix from a user's point of view.

**Comment by fschloesser** [comment]
Okay, so what would you do, let ipet crash with that warning/error or somehow skip the column in question?

File: ipet/evaluation/IPETEvalTable.py [hunk 1]
```diff
@@ -511,15 +511,25 @@ def tmpwrapper(*args):
                 if type(minval) in [int, float]:
                     result = numpy.maximum(result, minval)
                 else:
-                    result = numpy.maximum(result, minval.astype(result.dtype))
+                    comp = minval.astype(result.dtype)
+                    try:
+                        result = numpy.maximum(result, comp)
+                    except:
+                        logging.warning("When filling in the maximum, an error occurred for the column '{}':\n{}".format(self.getName(), self.attributesToStringDict()))
```
Comment (GregorCH) [inline ipet/evaluation/IPETEvalTable.py]: The message should contain the word "minimum".

Comment (GregorCH) [reply ipet/evaluation/IPETEvalTable.py]: @fschloesser

**Pull request merged**